MAIN/src/main/java/simplepets/brainsynder/commands/PetSubCommand.java

```diff
@@ -203,4 +205,12 @@ private org.bukkit.Color hex2Rgb(String hex) {
     public boolean isUsername (String string) {
         return (string.length() < 17) && (string.length() > 2) && string.replace("_", "").matches("[A-Za-z0-9]+");
     }
+
+    public void sendMessage(CommandSender sender, MessageOption messageOption) {
+        String message = MessageFile.getTranslation(messageOption);
+        if (!message.isEmpty()) {
+            sender.sendMessage(message);
+        }
+    }
+
```

Comment:
With respect to this, would this be better suited somewhere like the [MessageFile](https://github.com/brainsynder-Dev/SimplePets/blob/v5.0-Recode/MAIN/src/main/java/simplepets/brainsynder/files/MessageFile.java) class? So that not only are commands covered but event messages too.

Reply:
Yeah I agree! Would you like me to do this? :)

Reply:
Sure! If you'd rather have me or Brian take care of that though, let us know.

Reply:
Oh if you or Brian could take care of it that would be great!! 👍🏻 :D